### DISCUSSION TITLE
graphscope是否集成networkx的所有算法

### DISCUSSION THREAD

**Discussion by doingGod** [argoproj/argo-workflows#9428] [Q&A] [answered]
您好，
      我目前阅读了grpahscope的官方文档，了解到graphscope内置了networkx接口，但从文档的描述中发现graohscope只内置了部分networkx的算法接口，并没有内置networkx中一些常见的算法接口如介数中心性、紧密中心性算法等。如果我在开发中需要使用的这些未内置的算法，请问需要如何操作？

**Comment by terrytangyuan** [comment, marked as answer]
You are posting questions in the wrong repo.
